### DISCUSSION TITLE
Deployment doesn't work

### DISCUSSION THREAD

**Discussion by Wonderer11** [jekyll/jekyll#8977] [Q&A] [answered]
I want to deploy the latest Jekyll version on GitHub pages and I follow the tutorial https://jekyllrb.com/docs/continuous-integration/github-actions/ but the GitHub action doesn't work.
If anyone had successfully deploy Jekyll 4 on GitHub pages, can you tell me how?
Thank you.

**Comment by pranabdas** [comment, marked as answer]
@Wonderer11 Hello there, I faced some issues with GitHub pages automatic deploy, and needed to build and deploy via GitHub actions. I have created a minimal working example here <https://github.com/pranabdas/deploy-jekyll-github-action>, which is working for me.
